apps/sync/js/fxsync-webcrypto/fxsyncwebcrypto.js

```diff
@@ -0,0 +1,279 @@
+'use strict';
+
+/* global StringConversion, KeyDerivation, crypto */
+/* exported FxSyncWebCrypto */
+
+// WebCrypto-based client for Firefox Sync.
+
+var FxSyncWebCrypto = (function fxsync_webcrypto() {
+
+  const HKDF_INFO_STR = 'identity.mozilla.com/picl/v1/oldsync';
+
+  // constructor
```

Comment:
nit: in general, comments starting with Capital and ending with full stop please :)

Also, this comment is not that useful :P